api/src/main/java/org/openmrs/validator/PersonNameValidator.java

```diff
@@ -164,4 +180,31 @@ private String getFieldKey(String field, boolean arrayInd, boolean testInd) {
 		return testInd ? field : arrayInd ? "names[0]." + field : "name." + field;
 	}
 	
-}
+	
+	private boolean startsOrEndsWithWhitespace(String value) {
+		boolean startsOrEndsWithWhitespace = false;
+		if (value != null && StringUtils.isNotBlank(value)) {
```

Comment:
Isn't isNotBlank already checking for null?

Reply:
Just addressed your comment :)